livereveal/main.css

```diff
@@ -221,7 +221,7 @@
   font-size: 11px;
   background-color: #f7f7f7;
   color: #333;
-  box-shadow: 0 1px 0 rgba(0, 0, 0, 0.2), 0 0 0 2px #ffffff inset;
+  box-shadow: 0 1px 0 rgba(0, 0, 0, 0.2), 0 0 0 2px #fff inset;
```

Comment:
I changed nothing in less/css. This was a change that my less compiler introduced when I ran `npm run build`. Mine is lessc 2.5.3 on Mac OS.